### PR TITLE
Fix cron job run

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Fixing #322 
I tried with GitPython 1.0.1 I think there are issues with it and it was a 2015 release. So I resorted to version `2.1.8` that was there in our requirement.txt, however we should probably move to version 3.1.0

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -9,12 +9,13 @@ pipenv = "*"
...
 toml = "*"
 "delegator.py" = "*"
-gitpython = "==1.0.1"
+gitpython = "==2.1.8"
 pyyaml = "*"
 semver = "*"
 thoth-common = "*"
 thamos = "*"
 ogr = "*"
+gitdb2 = "==3.0.1"
```
Comment (fridex) [inline Pipfile]: Do we need this to be pinned?

**Comment by saisankargochhayat** [comment]
I have also created another pr with the latest gitpython version #324

File: Pipfile (hunk 1 above)
Comment (saisankargochhayat) [reply Pipfile]: That's what they suggest as after that it breaks.

Comment (saisankargochhayat) [reply Pipfile]: https://github.com/gitpython-developers/GitPython/issues/983
Alternative is to use latest gitpython version which I have made another PR for, we can close this if we use the latest version.

Comment (fridex) [reply Pipfile]: Let's use the more recent version then.

**Pull request closed without merging**